examples/zcEarly_scenario.hs

```diff
@@ -25,6 +27,6 @@ circuit uv oc zc gp_ack gn_ack gp gn =
     gnHandshake = handshake gn gn_ack
 
     initialState = initialise uv False <> initialise oc False
-
-    chargeFunc = ocFunc <> ocReact <> environmentConstraint <> circuitConstraint 
-                 <> gpHandshake <> gnHandshake <> initialState
+    
```

Comment:
I can see trailing spaces here and there. Can you set up your editor to remove them automatically (e.g. on Save)?